torch/csrc/lazy/backend/backend_interface.h

```diff
@@ -77,7 +77,7 @@ class TORCH_API BackendImplInterface {
       const std::string& device, c10::ArrayRef<std::string> devices) const = 0;
 
   virtual std::vector<ComputationPtr> Compile(
-      std::vector<ComputationPtr> instances) const = 0;
+      std::vector<ComputationPtr> instances, bool force_ltc_data=true) const = 0;
```

Comment:
i don't think `force_ltc_data` is a very descriptive name from the perspective of someone reading the backend interface.  It's not clear what the semantics of it are.  If it's false, what does it mean for the backend?

(I realize the name is borrowed from the 'coll.config' thing, but i'd rather not just cargo-cult it if it isn't aptly named

Reply:
Any suggestions for a better name? Maybe `in_mark_step`?